### PR TITLE
chore: Improve various design docs

### PR DESCRIPTION
**Pull request by VincentSmedinga**

### CODE REVIEW

File: storybook/storybook-docs/src/grid.stories.mdx [hunk 1]
```diff
@@ -148,7 +148,7 @@ Om hoeveel pixels het dan precies gaat is niet zo van belang.
 
 Leuk, al die flexibiliteit.
 Maar bij het ontwerpen van een scherm is het toch nodig een bepaald canvas te kiezen.
-Voor elk ontwerp wil je bepalen hoe het zich op ‘een dekstop’, ‘een tablet’ of ‘mobiel’ houdt.
+En voor elk ontwerp wil je bepalen hoe het zich op ‘een dekstop’, ‘een tablet’ of ‘mobiel’ houdt.
```
Comment (alimpens) [inline storybook/storybook-docs/src/grid.stories.mdx:151]:
```suggestion
En voor elk ontwerp wil je bepalen hoe het zich op ‘een desktop’, ‘een tablet’ of ‘mobiel’ houdt.
```

File: storybook/storybook-docs/src/typography.stories.mdx [hunk 2]
```diff
@@ -166,14 +164,15 @@ Vermijd het onderstrepen van tekst.
 Onderstreep alleen interactieve links.
 
 De lijndikte is 1/12 van de tekstgrootte, dus 2 pixels bij een standaard paragraaf van 24.
-De afstand tussen de onderstreping en de letterlijn is iets vergroot.
-Rond staartletters wordt de streep onderbroken voor betere leesbaarheid.
+De afstand tussen de onderstreping en de letterlijn is 3/8 van de tekstgrootte –
+meer dan we in onderstaande voorbeeld kunnen laten zien.
```
Comment (alimpens) [inline storybook/storybook-docs/src/typography.stories.mdx:168]: This implies a fluid styling imo, which currently isn't the case. The underline thickness is a static 0.125rem, and the distance between baseline and underline 0.375rem.

Comment (VincentSmedinga) [reply storybook/storybook-docs/src/typography.stories.mdx]: Ah yes. Adjusted.

File: storybook/storybook-docs/src/typography.stories.mdx [hunk 3]
```diff
@@ -165,15 +163,17 @@ Er zijn betere manieren om lange teksten nadruk te geven.
 Vermijd het onderstrepen van tekst.
 Onderstreep alleen interactieve links.
 
-De lijndikte is 1/12 van de tekstgrootte, dus 2 pixels bij een standaard paragraaf van 24.
-De afstand tussen de onderstreping en de letterlijn is iets vergroot.
-Rond staartletters wordt de streep onderbroken voor betere leesbaarheid.
+De lijndikte is 2 pixels.
+De afstand tussen de onderstreping en de letterlijn is 6 pixels –
+meer dan we in onderstaande voorbeeld kunnen laten zien.
+Als de muiswijzer over de link beweegt wordrt de lijn aan de bovenkant een pixel dikker.
```
Comment (alimpens) [inline storybook/storybook-docs/src/typography.stories.mdx:169]:
```suggestion
Als de muiswijzer over de link beweegt wordt de lijn aan de bovenkant een pixel dikker.
```

**Review by alimpens — APPROVED** [review]

**Pull request merged**